### PR TITLE
gpu: More tests and docs around Post Processing

### PR DESCRIPTION
**Pull request by spencer-lunarg**
Things discovered while working on https://github.com/KhronosGroup/Vulkan-ValidationLayers/issues/8688

### CODE REVIEW

File: docs/gpu_av_post_process.md [hunk 1]
```diff
@@ -36,33 +36,10 @@ The goal is that the `ValidateDescriptor()` function can be called the same from
...
-```
-[&set_a, &set_b, null, null]
-   |        |
-   |      [descriptor_0, descriptor_1, etc]
-   |
- [descriptor_0, descriptor_1, etc]
-```
-
-and the second draw would look like
+The pipelines contain the `OpVariable` SPIR-V information which is wrapped up in a `BindingVariableMap` pointer. We need this to know which variable was actually accessed in the case there are alias descriptors.
```
Comment (arno-lunarg) [inline docs/gpu_av_post_process.md:41]: New text needs to be proofread. Add a definition for `PostProcessDescriptorIndexSlot`. Ping me after!

Comment (spencer-lunarg) [reply docs/gpu_av_post_process.md]: done

File: docs/gpu_av_post_process.md [hunk 2]
```diff
@@ -36,33 +36,10 @@ The goal is that the `ValidateDescriptor()` function can be called the same from
 
 When tracking data for Post Process, we need both information from `vkCmdBindDescriptorSets` and `vkCmdBindPipeline` (or shader objects), but not all are present at the same time.
 
-At action command time we need to bind a `VkBuffer` that contains the Buffer Device Address to each descriptor set. If we have 4 descriptor set and the following command buffer stream:
+We have the `VkDescriptorSet` object contain a buffer with a `PostProcessDescriptorIndexSlot` for each descriptor in it. This "descriptor index slot" hold the information that is only known at GPU execution time (which index, which OpVariable, etc). For each action command, we create a LUT buffer that will point to the addresses for each post processing buffers.
```
Comment (arno-lunarg) [inline docs/gpu_av_post_process.md:39]:
```suggestion
The `VkDescriptorSet` object contains a buffer with a `PostProcessDescriptorIndexSlot` for each descriptor in it. This "descriptor index slot" holds information that can only be known at GPU execution time (which index, OpVariable, etc... are used). For each action command, we create a LUT mapping XXX to YYY.
```

**Review by arno-lunarg — APPROVED** [review]

**Comment by spencer-lunarg** [comment]
> CI Vulkan-ValidationLayers build # 18318 aborted.

this passed in  

> CI Vulkan-ValidationLayers build # 18316 aborted.

and just seems to be some unlucky networking issues

http://tcubuser.lunarg.localdomain:8080/job/Vulkan-ValidationLayers/18318/BITS=64,BUILD_MODE=Debug,label=Linux-Mesa-6800-rolling/console

**Pull request merged**